### ISSUE TITLE
[14.0] Errore in esportazione fattura con con linee note o sezione

### ISSUE THREAD

**Issue by matteoopenf** [OCA/l10n-italy#2492]
## Module
The name of the module that has a bug.
l10n_it_fatturapa_out

## Describe the bug
/opt/odoo/odoo/odoo/addons/base/models/qweb.py(338)_compiled_fn()
-> raise QWebException("Error to render compiling AST", e, path, node and etree.tostring(node[0], encoding='unicode'), name)

## To Reproduce
**Affected versions**:

1. Creare una fattura
2. Aggiungere una linea di note o sezioni
3. tentare l'esportazione

**Expected behavior**
la fattura venga esportata correttamente
**Additional context**
Add any other context about the problem here. (e.g. OS, Python version, ...)

**Comment by SimoRubi** [comment]
Guardando gli step per riprodurre mi pare duplicata di https://github.com/OCA/l10n-italy/issues/2463, puoi verificare se la soluzione proposta lì (https://github.com/OCA/l10n-italy/pull/2485) risolve anche questo problema?

**Comment by matteoopenf** [comment]
> Guardando gli step per riprodurre mi pare duplicata di #2463, puoi verificare se la soluzione proposta lì (#2485) risolve anche questo problema?

No, non risolve del tutto, la pr che segnali, ho già provato a integrarla, ma si rompe sull'xlpath per via della get che viene fatta

**Comment by matteoopenf** [comment]
> > Guardando gli step per riprodurre mi pare duplicata di #2463, puoi verificare se la soluzione proposta lì (#2485) risolve anche questo problema?
> 
> No, non risolve del tutto, la pr che segnali, ho già provato a integrarla, ma si rompe sull'xlpath per via della get che viene fatta

tuttavia è utile mergiare entrambe, però ho commentato anche l'altra che c''è un bug su quali linee si controllano al fine di esportare

**Comment by tafaRU** [comment]
Riapri pure nel caso il problema dovesse riproporsi.
